restcomm/restcomm.docs/sources-asciidoc/src/main/asciidoc/api/account-api.adoc

```diff
@@ -41,6 +41,25 @@ Accounts and sub-accounts are useful for things like segmenting phone numbers an
 |Password |A password that will be used to generate the AuthToken for the new Account resource.
 |==============================================================================================
 
+=== Account Migration to new Organization
+
```

Comment:
shoudl we document the different status codes (200->migrated,403->only superadmin, 400->nochildOgsuperadmin) and their meaning

Reply:
@jaimecasero good point, I will add this